src/languages/es.js

```diff
@@ -135,6 +135,9 @@ export default {
     reportActionsView: {
         beFirstPersonToComment: 'Sé el primero en comentar',
     },
+    reportActionsViewMarkerBadge: {
+        newMsg: ({count}) => `${count} nuevo${count > 1 ? 's' : ''} mensaje${count > 1 ? 's' : ''}`,
```

Comment:
I believe `nuevo` should come **after** `mensaje`

Reply:
Please tell me what is correct for new messages.

Reply:
"# new messages" => # mensajes nuevos
"# new message" => # mensaje nuevo